### ISSUE TITLE
xHE-AAC support?

### ISSUE THREAD

**Issue by prajaybasu** [microsoft/media-foundation#23]
Microsoft [licensed xHE-AAC almost a year ago](https://www.audioblog.iis.fraunhofer.com/fraunhoferiis-xheaac-microsoft).   
Is there any update on this? There seems to be no mention of xHE-AAC support anywhere in Windows or the AAC Decoder documentation.

**Comment by dianarodrigz** [comment]
Correct, the documentation is running behind, but the AAC decoder MFT should be able to decode xHE-AAC content on insider builds.
